prober/handler.go

```diff
@@ -137,6 +145,15 @@ func Handler(w http.ResponseWriter, r *http.Request, c *config.Config, logger lo
 	h.ServeHTTP(w, r)
 }
 
+func setTLSServerName(hostname string, module *config.Module) error {
```

Comment:
error is always nil? so doesn't make sense to return error in this case.

also, can we not inline this assignment?

Reply:
I removed the function and made the assignment inline in d207f64. Thanks!